compiler/src/dotty/tools/dotc/core/Phases.scala

```diff
@@ -295,6 +295,9 @@ object Phases {
     /** If set, implicit search is enabled */
     def allowsImplicitSearch: Boolean = false
 
+    /** If set equate Skolem types with underlying types */
+    def widenSkolems: Boolean = true
```

Comment:
I assume this should be false by default and only enabled in the rechecker?

Reply:
yes